HW_1/task_2.py

```diff
@@ -0,0 +1,11 @@
+"""
+Докажите, что выражение ¬(X ⋁ Y ⋁ Z) = ¬X ⋀ ¬Y ⋀ ¬Z истинно для всех значений предикат.
+
+Вывод: единственное значение типа bool (True либо False)
+"""
+res = []
+for x in [True, False]:
+    for y in [True, False]:
+        for z in [True, False]:
```

Comment:
Лучше использовать кортежи, а еще лучше множества, потому что проверка на вход в множество имеет константную сложность.
```suggestion
for x in True, False:
    for y in True, False:
        for z in True, False:
```